docs/source/using/troubleshooting.adoc

```diff
@@ -243,3 +243,70 @@ Workaround: Avoid installing packages or storing large files in the root filesys
 Instead, you can create a sub-directory in the *_mnt/sda1/_* persistent storage volume or define and mount xref:../using/host-folders.adoc#[host folders] that can share storage space between the host and the Minishift VM.
 
 If you want to perform development tasks inside the Minishift VM, it is recommended that you use containers, which are stored in persistent storage volumes, and reuse the xref:../using/docker-daemon.adoc#[Minishift Docker daemon].
+
+
+[[minshift-startup-check-failed]]
+== Minishift startup check failed
+
+While Minishift starts, it runs several startup checks to make sure that the Minishift VM and the OpenShift Cluster are able to start without any issues.
+If any configuration is incorrect or missing, the startup checks fail and Minishift does not start.
+
+If you want to force Minishift to start despite any failed checks, you can instruct Minishift to treat these errors as warnings as follows:
```

Comment:
Are these meant as examples, aka:

"If you want to force Minishift to start despite a failed check, you can instruct Minishift to treat these errors as warnings. For example to treat the KVM driver check as warning: ..."

If it is not an example, I am a bit confused, since it reads to me that depending on the driver, I just have to set say `minishift config set warn-check-xhyvek-driver true` and Minishift will start. 

I guess I am trying to figure out why the drivers checks are described in so much detail, before you start with: "The following sections describe the different startup checks.:" Would they not fall under there?